packages/fast-layouts-react/src/grid/grid.props.ts

```diff
@@ -30,6 +30,11 @@ export enum GridAlignment {
  */
 export type GridGutter = 0 | 2 | 3 | 6 | 12;
 
+/**
+ * The CSS grid display style
+ */
+export type GridDisplay = "grid" | "-ms-grid";
```

Comment:
@kim-msft can you make sure this is being exported from the `index.ts` file?

Reply:
Updated - thank you!